src/main/steps/respondent/legal-jurisdiction-of-the-courts/template.njk

```diff
@@ -11,19 +11,26 @@
       <li>{{ line | safe }}</li>
     {% endfor %}
   </ul>
-  {% set jurisdictionsMoreDetailsTemplate %}
-    {% if jurisdictionsMoreDetails.length == 1 %}
-      {{ jurisdictionsMoreDetails[0] | safe }}
-    {% else %}
-      {% for jurisdictionItem in jurisdictionsMoreDetails %}
-        <h3 class="govuk-heading-s govuk-!-margin-bottom-1">{{ jurisdictionItem.heading }}</h3>
-        <p class="govuk-body govuk-!-margin-bottom-2">{{ jurisdictionItem.body | safe }}</p>
-      {% endfor %}
-    {% endif %}
+  {% set whatThisMeansContent2 %}
+    <h3 class="govuk-heading-m">{{ heading1 }}</h3>
+    <p class="govuk-body">{{ habitualResidenceText | safe }}</p>
+
+    <h3 class="govuk-heading-m">{{ heading2 }}</h3>
+    <p class="govuk-body">{{ domicileText | safe }}</p>
+
+    <h3 class="govuk-heading-m">{{ heading3 }}</h3>
+    <p class="govuk-body">{{ residualJurisdictionLine1 }}</p>
+    <p class="govuk-body">{{ residualJurisdictionLine2 }}</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>{{ residualJurisdictionListItem1 }}</li>
+      <li>{{ residualJurisdictionListItem2 }}</li>
+      <li>{{ residualJurisdictionListItem3 }}</li>
+    </ul>
+    <p class="govuk-body">{{ residualJurisdictionLine3 }}</p>
```

Comment:
this might be a content change that I'm not aware of, but the way the jurisdictionMoreDetails worked before was that the text appears dynamically based on the jurisdiction connection made. Is that something that's suppose to change?

Reply:
Hi Lara, so i added all this content as it was not displaying when you click on "What this means". Here is the google docs that i was using for reference: https://docs.google.com/document/d/1j2a0Zd-Oa1CtsU6w_setnr1vKlya5jYfWR_ThlM0PQc/edit#heading=h.ijki9xx42mrf 

Also here is a screenshot of the issue on the master branch. 
![Screenshot 2022-06-07 at 15 36 49](https://user-images.githubusercontent.com/93586197/172409416-16b0e52b-5638-494f-a952-aa6e3c5dac19.png)

Reply:
thanks Lara, yeah i'll have a chat with Michael to get more clarification.

Reply:
Hi Lara, just a small observation regarding the content for "what this means" section, In the master branch, the following line "jurisdictionsMoreDetails: jurisdictionMoreDetailsContent(userCase?.connections, isDivorce, true).text," the "true" is for show all connection types. That would mean that the habitual, domicile and residual content should all show up under "what this means" dropdown? This is the file am referring to:  https://github.com/hmcts/nfdiv-frontend/blob/master/src/main/steps/respondent/legal-jurisdiction-of-the-courts/content.ts